new.go

```diff
@@ -3,19 +3,22 @@
 package genji
 
 import (
+	"context"
+
 	"github.com/genjidb/genji/database"
 	"github.com/genjidb/genji/document/encoding/msgpack"
 	"github.com/genjidb/genji/engine"
 )
 
 // New initializes the DB using the given engine.
-func New(ng engine.Engine) (*DB, error) {
-	db, err := database.New(ng, database.Options{Codec: msgpack.NewCodec()})
+func New(ctx context.Context, ng engine.Engine) (*DB, error) {
+	db, err := database.New(ctx, ng, database.Options{Codec: msgpack.NewCodec()})
 	if err != nil {
 		return nil, err
 	}
 
 	return &DB{
-		DB: db,
+		DB:      db,
+		Context: ctx,
```

Comment:
We should not pass that context here as it's supposed to be used for controlling the opening of the database.
See https://github.com/genjidb/genji/blob/master/rfcs/001_context.md#open-and-new

Reply:
Good call!
```suggestion
		Context: context.Background(),
```